web/app/components/Layout/Header.jsx

```diff
@@ -195,9 +195,9 @@ class Header extends React.Component {
             </div>
         ) : null;
 
-        let tradeLink = this.props.lastMarket && active.indexOf("market/") === -1 ?
```

Comment:
There needs to be a default value (USD_BTS) for new users so you can't remove that. I suspect a simple fix could be to remove the `&& active.indexOf("market/") === -1` condition, as that's what's directing you to the default value when you're already in the exchange page.